### ISSUE TITLE
Ecovim initialization with Lazyman hangs

### ISSUE THREAD

**Issue by doctorfree** [ecosse3/nvim#145]
[Lazyman](https://github.com/doctorfree/nvim-lazyman) performs a command line initialization of supported Neovim configurations. Ecovim is one of the supported configurations. Unfortunately, Lazyman does not support interactive initialization.

The Ecovim initialization performed with `nvim --headless +Lazy! sync +qa` is hanging when prompted with the following:

```
dap_repl parser already available: would you like to reinstall ? y/n:
```

I believe this is due to the duplicate Tree-Sitter install/update in `lua/config/plugins.lua`:

```lua
	{
		"LiadOz/nvim-dap-repl-highlights",
		config = true,
		dependencies = {
			"mfussenegger/nvim-dap",
		},
		build = {
			":TSInstall dap_repl",
			":TSUpdate dap_repl",
		},
	},
```

It's one of my two or three favorite configs but  I am removing Ecovim from Lazyman until this is resolved. Maybe I can figure out a way to work around this.

**Comment by doctorfree** [comment]
I removed Ecovim from Lazyman but then I felt so bad about it that I wrote a timeout function for the `lazyman` command. I'm still testing but it looks like I will be able to reinstate Ecovim by using a timeout function to run Neovim and terminate it after some specified time if still running.

What this will end up doing if Ecovim continues to require interaction during initialization is to interrupt the initialization process at the point it emits the prompt. Fortunately, it appears that Ecovim then works fine since the remaining initialization was Tree-Sitter and Mason installs which happen the next time it is run.

I will close this issue as resolved on my end but I would encourage you to avoid any requirement for interaction during initialization.
